Tensile/KernelWriterAssembly.py

```diff
@@ -6137,10 +6220,24 @@ def mfmaIter(self, kernel, m, innerUnroll, tail=False):
         for a in range(0, kernel["MIWaveTileA"]):
           for iui in range(0, innerUnroll):
             aStr = vgpr("ValuA_X%u_I%u+%u+%u" % (m, iui, a*vgprPerInput, bk), 1)
+            if kernel["DirectToVgprA"]:
+              # overwrite aStr for DirectToVgprA
+              tmp   =  0
+              tmp   = (tmp + iui) * kernel["MIWaveTileA"]
+              tmp   = (tmp + a) * vgprPerInput + bk
+              tmp   += vregSetIdx * numVgprPerBlockA
```

Comment:
Not critical, but some of this calculation can be moved out as it's duplicated a bunch of times in this section of code. Could pre-calculate partial index and add in a*vgprPerInput+bk after

Reply:
I simplified this part and combined some parts.